docs: Add 4-pin DIN connector info to modules

jwwojak opened the pull request:
# Overview

The Temperature Module and the Heater-Shaker use a 4-pin DIN power connector that has a unique shape (partly rounded and flat). Sometimes users don't notice this and try to force the connector into the module. This can damage the module and connector. Adding a new row to the specifications tables for each of these modules to show and describe this connector. It may not prevent this problem, but at least we'll have documented it.

This is for the Flex manual only. Modules chapter. Reference: RTC-685. Separate tickets will update quickstarts and instruction manuals.


## Test Plan and Hands on Testing

Check table code and text for accuracy. I've not been using a relative path to the images file and will start doing that now.

## Changelog

Adds this row and new image to specifications table:

```
<td><strong>Power adapter connector</strong></td>
<td>
   4-pin DIN<br>
   <img src="../images/module-power-connector.png" alt="4-pin DIN connector with flat side" width="40%"><br>
   Align the connector's flat side with the flat side of the module's power port.
</td>
```
Also changes 2 markdown tables to HTML to better handle this complexity.

## Review requests

Putting a border around the image in the cell may make it look odd. Maybe float it? Not really sure what to do here. Looks OK as is, IMO, but can change if there's another way. It conveys the idea.

## Risk assessment

Low. Documentation only.

jwwojak committed:
Start PR: convert table and add new image

Make table HTML.

Upload connector image. No .svg available. We're going to have to work with this. Updated from Jackson.

jwwojak committed:
Add connector infor specifications tables

For the Heater-Shaker and Temperature Modules, add new row in the specification tables for each.

This shows the 4-pin DIN connector and includes text about alignment for proper connection. Same for both modules.

ecormany commented:
I don't know if the image in the specifications table does enough to put this info in a prominent place, and it also breaks up the table formatting significantly. How about putting this information in this section of the chapter:
 

> In general, to install a module caddy:
> 
> 1.  Remove any deck slots from the location where the module will go.
> 2.  Seat the module into its caddy and tighten its anchors.
> 3.  Route the module power and USB cables through the side covers, up through the empty deck slot, and attach them to the module.
> 4.  Seat the module caddy into the slot and screw it into place.
> 
> For exact installation instructions, consult the Quickstart Guide or Instruction Manual for the specific module. Cable connections and method of attachment to the caddy vary by module.

We could put a warning admonition, including the image, in one of a couple places:
- Embedded in step 3, which is about connecting cables.
- At the end, where we mention that different modules have different cables.

The warning text would be something like this:
```
!!! warning
    The Heater-Shaker Module and Temperature Module have asymmetrical 4-pin DIN power connectors. Align the connector's flat side with the flat side of the module's power port. 
    [image]
    _Do not_ force cable connections, or you may damage the module.
```

jwwojak committed:
Ignore this commit

Committing to save it. More coming shortly.

jwwojak committed:
Add power cable warning to caddy section

Adds text, bullets, and image that explain the odd DIN power connector used by the heater-shaker and temp mods. Revisions from review comments.

jwwojak commented:
Removed text and images from module specs tables. Replaced w/ a warning at the end of the module caddy section.

<img width="677" height="478" alt="Screenshot 2025-07-18 at 2 58 26 PM" src="https://github.com/user-attachments/assets/83a76160-6e98-47ad-ba4e-d5e520016b4d" />

ecormany reviewed (approved):
Looks good with the new warning!
